fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -404,9 +403,7 @@ public void updateProperties(final IdentifierConverter<Resource, FedoraResource>
         final Model maybeModel = idTranslator.reverse().convert(this).getModel();
         final Model context = maybeModel == null ? createDefaultModel() : maybeModel;
 
-        final Deskolemizer deskolemize = new Deskolemizer(idTranslator, context);
-
-        final Model model = originalTriples.withThisContext(originalTriples.transform(deskolemize)).asModel();
```

Comment:
This would have been simpler as `final Model model = originalTriples.map(deskolemize).asModel();`